rtsp.go

```diff
@@ -179,9 +207,13 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 	// if SPS and PPS are present into the SDP, send them to the decoder
 	if format.SPS != nil {
 		rc.rawDecoder.decode(format.SPS)
+	} else {
+		rc.logger.Warn("no SPS found in H264 format")
```

Comment:
Can h264 image decoding succeed if either SPS or PPS are not set?

Reply:
I am not 100% sure, but I would assume the decoder will throw errors and fail if parameter sets are not provided.

If the param sets are not forwarded through RTP/TCP payloads, I think it is still potentially possible to pull data from SDP description and manually create a SPS message.  Not sure how PPS would be handled in that fallback case.

I have not seen this be an issue yet while testing. Added those verbose logs in so users are aware of missing param sets.